ui/app/AppLayouts/Chat/controls/Contact.qml

```diff
@@ -33,8 +33,6 @@ Rectangle {
 
     visible: isVisible && (isContact || isUser)
     height: visible ? 64 : 0
-    anchors.right: parent.right
-    anchors.left: parent.left
```

Comment:
Just to be sure, you checked all places where `Contact` was used and added the width or anchors there instead?

Reply:
Yes, i've checked other places with Contacts